README.md

```diff
@@ -14,6 +14,14 @@ Please check the [**Contribution
 Guidelines**](https://parallaxsecond.github.io/parsec-book/contributing/index.html) to know more
 about the contribution process.
 
+# History
+
+This repository is based on [this original PR on rust-pkcs11](https://github.com/mheese/rust-pkcs11/pull/43).
+Read the PR discussion for more information.
+
+Thanks to Joe Ellis (@joechrisellis) and Ionuț Mihalcea (@ionut-arm) for their early work on the
+crate.
```

Comment:
At least add yourself as well - "Joe Ellis, Ionut Mihalcea, and Hugues de Valon developed the early prototype...". Seems weird to only mention us two when it's not on your personal repo.

Reply:
Alternatively just don't mention anyone - `Parsec developers created ....` though that's kinda superfluous

Reply:
hmm yeah I wanted to credit those involved in this... I will just add them as co-author of this specific commit.

Reply:
(I should have done that for the first commit but I forgot)